pkg/karmadactl/register.go

```diff
@@ -204,6 +205,8 @@ type CommandRegisterOption struct {
 	// ClusterRegion represents the region of the cluster locate in.
 	ClusterRegion string
 
+	EnableCertRotation bool
```

Comment:
Please add a comment here.
(I'm surprised that the static check(golint) does not work now.)